### PR TITLE
Radon complete pool

### PR DESCRIPTION
**Pull request by stefanwebb**
Fixed bug of cases where Cauchy distribution was used for standard deviation

### CODE REVIEW

File: pyro_models/arm/radon_complete_pool.py [hunk 1]
```diff
@@ -35,7 +35,7 @@ def model(data, params):
     # initialize transformed parameters
     # model block
 
-    sigma =  pyro.sample("sigma", dist.Cauchy(0., 2.5))
+    sigma =  pyro.sample("sigma", dist.HalfCauchy(0., 2.5))
```
Comment (jpchen) [inline pyro_models/arm/radon_complete_pool.py]: `HalfCauchy` is centered at 0 so i think it should be `dist.HalfCauchy(2.5)`

Comment (stefanwebb) [reply pyro_models/arm/radon_complete_pool.py]: Ah yes! Let me change that

**Comment by stefanwebb** [comment]
Okay, should be good to go!

File: pyro_models/misc/ma2.py [hunk 2]
```diff
@@ -24,9 +24,9 @@ def model(data, params):
     y = data["y"]
 
     # model block
-    mu =  pyro.sample("mu",  dist.HalfCauchy(0., 2.5))
-    theta =  pyro.sample("theta",  dist.HalfCauchy(0., 2.5).expand([2]))
-    sigma =  pyro.sample("sigma",  dist.HalfCauchy(0., 2.5)).abs()
+    mu =  pyro.sample("mu",  dist.HalfCauchy(2.5))
+    theta =  pyro.sample("theta",  dist.HalfCauchy(2.5).expand([2]))
+    sigma =  pyro.sample("sigma",  dist.HalfCauchy(2.5)).abs()
```
Comment (jpchen) [inline pyro_models/misc/ma2.py]: you can remove this `abs()`

Comment (stefanwebb) [reply pyro_models/misc/ma2.py]: Done!

**Pull request merged**